src/components/AsyncTable/Column.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { createElement, ReactNode } from 'react';
+import { createElement, ReactNode } from 'react';
```

Comment:
I started removing these React imports (as they aren't needed anymore) but there are like 800 more so we can do that in a separate PR 😂